packages/graphql/test/websocket_test.dart

```diff
@@ -159,6 +165,47 @@ Future<void> main() async {
         ),
       );
     });
+    test('disconnect via dispose', () async {
+      // First wait for connection to complete
+      await expectLater(
+        socketClient.connectionState.asBroadcastStream(),
+        emitsInOrder(
+          [
+            SocketConnectionState.connecting,
+            SocketConnectionState.connected,
+          ],
+        ),
+      );
+
+      // We need to begin waiting on the connectionState
+      // before we issue the command to disconnect; otherwise
+      // it can reconnect so fast that it will be reconnected
+      // by the time that the expectLater check is initiated.
+      await overridePrint((_) async {
+        Timer(const Duration(milliseconds: 20), () async {
+          await socketClient.dispose();
+        });
+      })();
+      // The connectionState BehaviorController emits the current state
```

Comment:
```suggestion
     
       // The connectionState BehaviorController emits the current state
```
I preferer a space here